static/app/components/events/interfaces/crashContent/exception/utils.tsx

```diff
@@ -20,6 +22,47 @@ export function isFrameFilenamePathlike(frame: Frame): boolean {
   );
 }
 
+// Detects URLs in text and renders them with anchor tags
+export function Linkify({exceptionText}: {exceptionText?: string}): React.ReactElement {
```

Comment:
@mdtro does this function seem safe to you? Just want to make the link we show is always the raw URL and that someone can't use this to get a person to unknowingly click on a malicious link.